### DISCUSSION TITLE
I'm trying to construct a valued ERGM model using `ergm.count`, but the runtime is excessively long.

### DISCUSSION THREAD

**Discussion by 5IMX** [statnet/ergm#556] [Q&A]
I tried to build a valued ergm model using `ergm.count`. My data is a 284x284 matrix. I set the number of iterations to 20, but it has been running for almost a week without completion. I would like to ask whether this is normal, if there is a problem with the model construction, or if my data is indeed too large.

etzv=ergm(tzvNet ~ sum(pow=1) + mutual("min") + nodeicovar(transform="sqrt") + nodeocovar(transform="sqrt") + transitiveweights("min", "max", "min")
          + nodefactor("capital", levels=-1) + nodecov("gdp") + nodecov("dl") + nodecov("bci") + nodecov("pop") + nodecov("policy") + nodecov("government") + edgecov(distance) + edgecov(history) + edgecov(region), response="count", reference=~Binomial(4),control=control.ergm(MCMLE.maxit=50))
@krivit Thank you！

**Comment by krivit** [comment]
It's hard to say without more information or output logs. Can you please provide those?
